tianracer_slam/param/2d_scan.lua

```diff
@@ -19,11 +19,12 @@ options = {
   map_builder = MAP_BUILDER,
   trajectory_builder = TRAJECTORY_BUILDER,
   map_frame = "map",
-  tracking_frame = "base_footprint",
+  tracking_frame = "imu_link",
   published_frame = "odom",
   odom_frame = "odom",
   provide_odom_frame = false,
   publish_frame_projected_to_2d = true,
+  use_pose_extrapolator = on,
   use_odometry = true,
```

Comment:
Do imu readings compensate the pose when use_odometry is set to true? I remember previously IMU will take effect if use_odomety

Reply:
AFAIK, imu will be used to extrapolate rotation while odometry will be used to extrapolate translation in PoseExtrapolator.

Odometry will be used to predict rotation only if the IMU is not